platform/ios/PolyPodApp/PolyPodTests/Info.plist

```diff
@@ -11,6 +11,14 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
+	<key>NSAppTransportSecurity</key>
```

Comment:
@GhenadiePusca Did I understand this correctly that we (don't need to, but) want to set the ATS http-exception only in the PolyPodTests or also in the actual app?

In there I only found:
```
<key>NSAppTransportSecurity</key>
	<dict>
		<key>NSAllowsArbitraryLoads</key>
		<true/>
	</dict>
```
and was wondering where it's disabled

Reply:
In the actual app the ATS is disabled.
This setting will allow all loads.
```
<key>NSAllowsArbitraryLoads</key>
		<true/>
```
When it is wanted to have ATS enabled for all domains, but disabled for some domain, then exceptions are used. As it is right now, everything is allowed, no need to add exceptions.

I think we want to leave as is for now. No need to add additional configuration.

Reply:
Ok makes sense, thanks for clarifying!